src/plugins/content_management/README.md

```diff
@@ -0,0 +1,236 @@
+      getTargetArea: () => `${HOME_PAGE_ID}/${SECTIONS.SERVICE_CARDS}`,
+      getContent: () => ({
+        id: 'visualization_content_id',
+        kind: 'visualization', // with `visualization` kind
+        order: 1000,
+        input: {
+-          kind: 'static',
+-          id: 'c0ba29f0-eb8f-11ed-8e00-17d7d50cd7b2' // the visualization id
++          kind: 'dynamic',
++          get: () => Promise.resolve('c0ba29f0-eb8f-11ed-8e00-17d7d50cd7b2') // the visualization id
+        }
+      }),
+    });
+  }
+}
+```
+
+Add a saved dashboard to a `dashboard` section **statically**
+```typescript
+export class MyPlugin implements Plugin {
+  public start(core, { contentManagement }) {
+    contentManagement.registerContentProvider({
+      id: `dashboard_content_provider`,
+      getTargetArea: () => `${HOME_PAGE_ID}/${SECTIONS.SERVICE_CARDS}`,
+      getContent: () => ({
+        id: 'dashboard_content_id',
+        kind: 'dashboard', // with `dashboard` kind
+        order: 1000,
+        input: {
+          kind: 'static',
+          id: 'c39012d0-eb7a-11ed-8e00-17d7d50cd7b2' // the saved dashboard id
+        }
+      }),
+    });
+  }
+}
+```
+
+Similarly, you can add a saved dashboard to a `dashboard` section **dynamically**
+```diff
+export class MyPlugin implements Plugin {
+  public start(core, { contentManagement }) {
+    contentManagement.registerContentProvider({
+      id: `dashboard_content_provider`,
+      getTargetArea: () => `${HOME_PAGE_ID}/${SECTIONS.SERVICE_CARDS}`,
+      getContent: () => ({
+        id: 'dashboard_content_id',
+        kind: 'dashboard', // with `dashboard` kind
+        order: 1000,
+        input: {
+-          kind: 'static',
+-          id: 'c39012d0-eb7a-11ed-8e00-17d7d50cd7b2' // the saved dashboard id
++          kind: 'dynamic',
++          get: () => Promise.resolve('c39012d0-eb7a-11ed-8e00-17d7d50cd7b2') // the saved dashboard id
+        }
+      }),
+    });
+  }
+}
+```
+
+You can also add custom components to a `dashboard` section
+```typescript
+export class MyPlugin implements Plugin {
+  public start(core, { contentManagement }) {
+    contentManagement.registerContentProvider({
+      id: `custom_content_provider`,
+      getTargetArea: () => `${HOME_PAGE_ID}/${SECTIONS.SERVICE_CARDS}`,
+      getContent: () => ({
+        id: 'custom_content_id',
+        kind: 'custom', // with `dashboard` kind
+        order: 1000,
+        render: () => <MyComponent />
+      }),
+    });
+  }
+}
+```
+
+#### `custom` section
+If the existing pre-defined sections do not meet your needs, you could use `custom` section to customize the rendering of the contents,
+a custom section is typically defined with `kind: 'custom'` and a `render` function:
+```typescript
+{
+  id: SECTIONS.RECENTLY_VIEWED,
+  order: 2000,
+  title: 'Recently viewed',
+  kind: 'custom',
+  render: (contents) => (
+    <>
+      {contents.map((content) => content.kind === 'custom' ? content.render() : null)}
+    </>
+  ),
```

Comment:
Nit: we can mention that the `contents` has already been sorted by order field in ascending order.

Reply:
Sounds good, I'll add it